### PR TITLE
Modified README and environment files as per onboarding notes in issue 852

### LINKED ISSUE #852: A few updates to documentation for onboarding/README/environment file
Here are a few suggestions to update the `README` file and the `environment*.yml` files (credit @york-stsci):

- Make it clear that the project supports both python 3.7 and 3.8, and users/contributors may install either environment.
- Add a `name` attribute to the conda environment files, removing the need to specify the `-n` flag (as described in `README`)

### PR DESCRIPTION
**Pull request by york-stsci**
Closes #852

- Adds names to both (python 3.7 and 3.8) environment.yml files
- Edits the README to include that information, and specify that both environment files are options
- Edits the environment.yml files to do a local package installation, including the `-e` flag
- Edits the .gitignore file to add the jwql/website/apps/jwql/static/* to avoid the symbolic links being included

### CODE REVIEW

**Comment by york-stsci** [comment]
I realized that the  jwql/website/apps/jwql/static directory contained more than the symbolic links, so instead of ignoring everything in it, the .gitignore now specifically ignores the four symbolic link files in it.

**Comment by york-stsci** [comment]
@bourque I believe this one is now ready.

File: environment_python_3_7.yml [hunk 1]
```diff
@@ -1,42 +1,66 @@
...
+# $ conda env create -n <custom name> -f environment_python_3_7.yml
+#
+# Run the following command to activate the environment:
+# $ source activate jwql-3.7
+#
+# To deactivate the environment run the following command:
+# $ source deactivate
+#
+# To remove the environment entirely, run the following command:
+# $ conda env remove -n jwql-3.7
```
Comment (bourque) [inline environment_python_3_7.yml:16]: Thank you for adding these docs, this is great!

File: environment_python_3_7.yml [hunk 2]
```diff
@@ -1,42 +1,66 @@
...
+    - crds==11.5.2
+    - jwst==1.3.3
+    - pysiaf==0.11.0
+    - pysqlite3==0.4.3
+    - stdatamodels==0.2.4
+    - stsci_rtd_theme==0.0.2
+    - git+https://github.com/spacetelescope/jwst_reffiles
+    
+    # Current package
+    - -e .
```
Comment (bourque) [inline environment_python_3_7.yml:66]: Great idea!

**Review by bourque — APPROVED** [review]
Thanks @york-stsci, this is great!  Nice work, it all looks good.  I took this PR as an opportunity to update a few more things in the README.  This is good to merge pending successful CI builds.

**Pull request merged**